src/main/java/lotto/domain/WinAmount.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain;
+
+public enum WinAmount {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
와 같은 표현으로 가독성을 더 확보할 수 있습니다!